api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -21,7 +21,7 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class BillingAlertsService {
```

Comment:
Renaming this because alerting is a subset of what this does: handle free tier billing information